### PR TITLE
fix #37349 kompkimi.ru

### LINKED ISSUE #37349: kompkimi.ru
### Issue URL (Ads)
[https://kompkimi.ru/windows10/kak-ubrat-nadpis-aktivatsiya-windows-10](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fkompkimi.ru%2Fwindows10%2Fkak-ubrat-nadpis-aktivatsiya-windows-10)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/37349

### CODE REVIEW

**Comment by Alex-302** [comment]
ad leftover
`||kompkimi.ru/wp-content/uploads/*/direct.php`
+
`||info.kinoclub77.ru^` - advertising sub-domain
image

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -34,7 +34,8 @@ brestcity.com##.BlockTop1
 brestcity.com###page > div#wrapper~a[rel="nofollow"] > img
 mysitem.ru##.entry-content > .custom
 pornotor.org##center > a[rel="nofollow"] > img
-kompkimi.ru##.widget_sitemoney
+||kompkimi.ru/wp-content/uploads/*/direct.php
+||info.kinoclub77.ru^$domain=kompkimi.ru
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: remove `$domain=kompkimi.ru` and move rule to adservers_firstparty.txt

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (tvinzz) [reply RussianFilter/sections/specific.txt]: @Alex-302, done

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -12,6 +12,7 @@ ligainsider.de###popup_extra
 kickassanime.io###notes
 ! facebook.com - reactions in live-video even when disabled via button
 facebook.com###fbVideoStreamingReactionsTray
+kompkimi.ru###print-content > .box_author > .box-option > .p_bold
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: why you block that element?
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 2 above)
Comment (tvinzz) [reply AnnoyancesFilter/sections/annoyances.txt]: @Alex-302, removed redundant rule

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**